### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the content of "Sign in with your phone or token device" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1496]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5. In" Azure Login" Enter the "email or phone number" in the edit box and select Next button.
6. Tab to and select "Sign in with a user name and password instead" link.
7. Enter the required password and select "sign in" button.
8. Tab to and select "sign in with your phone or token device" link .
9. Observe Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA are not announcing the content present in "Sign in with your phone or token device" dialog 

**Expected Result:**
Jaws and NVDA should announce the content of the "Sign in with your phone or token device" dialog 

Jaws and NVDA should announce as
Azure Login Dialog
tab
Sign in with PIN or smart card instead link
tab 
Sign in with your phone or token device

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![51aaaa4d](https://user-images.githubusercontent.com/22956130/36136535-0d34146c-10b7-11e8-9105-a11ddea9ffeb.jpg)
![azure login dialog-jaws](https://user-images.githubusercontent.com/22956130/36136540-10e2b7c6-10b7-11e8-974b-5dd1702700f0.jpg)

**Comment by Eskibear** [comment]
According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749946
